config/settings.yml

```diff
@@ -97,6 +97,7 @@
     :purge_window_size: 10000
 :ems:
   :ems_redhat:
+    :resolve_ip_addresses: true
```

Comment:
Can this be commented? I didn't see comments in this file, but if possible, I'd like to write a comment explaining the meaning. Something like this:

```yaml
#
# Indicates if the oVirt provider should try to convert IP addresses
# to host names, using a reverse DNS lookup if required. This is
# needed because since version 4 of oVirt connections that use
# an IP address directly are rejected. Change to 'false' only if you
# have explicitly configured the oVirt engine to use IP addresses
# and you don't have a working DNS configuration.
#
:resolve_ip_addresses: true
```

Assuming comments are allowed, would that be enough?